### DISCUSSION TITLE
Why can't I get a response to my takedown notice？

### DISCUSSION THREAD

**Discussion by behappy628** [community/community#118634] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have a code warehouse that has not been copied by others, and now I want to remove this code warehouse. I have submitted many removal applications, but the official has not given any reply. This matter is very important to me, I may lose my job because this warehouse is not available, I submit new takedown notice every day, but I don't know why I can't receive any reply. The Ticket ids of the removal applications I mentioned before are 2722424, 2720605 and 2681048 respectively. May I ask why the removal applications submitted ten days ago have not been replied?

**Comment by ebndev** [comment]
Hi [wangxiang1](https://github.com/wangxiang1),

Thank you for your patience and being a part of the Github community. We are unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you.
